### PR TITLE
[BUILDING] Design Contributino Game

### PR DESCRIPTION
**Pull request by MaxHillebrand**
This adds the new design contribution game, introduced here https://github.com/zkSNACKs/Meta/issues/76.

### CODE REVIEW

File: docs/building-wasabi/ContributionGame.md [hunk 1]
```diff
@@ -38,3 +38,10 @@ This enabled many more reviewers to check out the latest additions to the softwa
 Also there was a [contribution game](https://github.com/zkSNACKs/WalletWasabi/issues/2631) where any reviewer who provided a thorough report and notified the developers of bugs was eligible for a bounty of in total `0.3 bitcoin`.
 
 The total bounty ended up being distributed to 6 contributors, see the full results [here](https://github.com/zkSNACKs/WalletWasabi/issues/2631#issuecomment-565746074).
+
+## Merchandise design
+
+In order find a beautiful design for Wasabi t-shirts, hoodies and stickers, there was a contribution game for any artist who created them.
```
Comment (yahiheb) [inline docs/building-wasabi/ContributionGame.md]:
```suggestion
In order to find a beautiful design for Wasabi t-shirts, hoodies and stickers, there was a [contribution game for any artist](https://github.com/zkSNACKs/Meta/issues/76) who created them.
```

File: docs/building-wasabi/ContributionGame.md [hunk 2]
```diff
@@ -38,3 +38,10 @@ This enabled many more reviewers to check out the latest additions to the softwa
...
+The art should reflect privacy in general, Bitcoin and Wasabi specifically.
+There was a total bounty of up to 0.1 bitcoin.
+See the results [here](https://github.com/zkSNACKs/Meta/issues/76).
```
Comment (yahiheb) [inline docs/building-wasabi/ContributionGame.md]: This link should be changed later.

Comment (MaxHillebrand) [reply docs/building-wasabi/ContributionGame.md]:
```suggestion
```

File: docs/building-wasabi/ContributionGame.md [hunk 3]
```diff
@@ -38,3 +38,9 @@ This enabled many more reviewers to check out the latest additions to the softwa
 Also there was a [contribution game](https://github.com/zkSNACKs/WalletWasabi/issues/2631) where any reviewer who provided a thorough report and notified the developers of bugs was eligible for a bounty of in total `0.3 bitcoin`.
 
 The total bounty ended up being distributed to 6 contributors, see the full results [here](https://github.com/zkSNACKs/WalletWasabi/issues/2631#issuecomment-565746074).
+
+## Merchandise design
+
+In order to find a beautiful design for Wasabi t-shirts, hoodies and stickers, there was a [contribution game for any artist](https://github.com/zkSNACKs/Meta/issues/76) who created them.
```
Comment (yahiheb) [inline docs/building-wasabi/ContributionGame.md]:
```suggestion
In order to find a beautiful design for Wasabi t-shirts, hoodies and stickers, there was a [contribution game for artists](https://github.com/zkSNACKs/Meta/issues/76).
```

File: docs/building-wasabi/ContributionGame.md [hunk 4]
```diff
@@ -38,3 +38,9 @@ This enabled many more reviewers to check out the latest additions to the softwa
...
+The art should reflect privacy in general, Bitcoin and Wasabi specifically.
```
Comment (yahiheb) [inline docs/building-wasabi/ContributionGame.md]:
```suggestion
The art reflect privacy in general, Bitcoin and Wasabi specifically.
```

Comment (MaxHillebrand) [reply docs/building-wasabi/ContributionGame.md]: Your suggestion doesn't make sense... why remove the `should`?

Comment (yahiheb) [reply docs/building-wasabi/ContributionGame.md]:
```suggestion
The subject was about privacy in general, Bitcoin and Wasabi specifically.
```

**Comment by yahiheb** [comment]
Shouldn't this have waiting until the contribution game takes place and then merge this PR?

**Comment by MaxHillebrand** [comment]
I think it's ok to add now, as then we can link to this docs for others who want to participate in it.

it is already now written in past tense, so that we do not have to change it afterwards... that is not optimal, but better for maintenance.

**Comment by yahiheb** [comment]
Well I think we should simply point them to https://github.com/zkSNACKs/Meta/issues/76.
What if it never takes place? or something changes?

**Pull request merged**